packages/zosjobs/src/SearchJobs.ts

```diff
@@ -0,0 +1,162 @@
+                                replyBuffer = replyBuffer + "Job Name: " + job.jobname + " Job Id: " + job.jobid +
+                                    " Spool file: " + spoolFile.ddname + "(ID #" + spoolFile.id + " Step: " +
+                                    spoolFile.stepname + " ProcStep: " + spoolFile.procstep +")" + "\n";
+
+                            } else {
+                                replyBuffer = replyBuffer + "Job Name: " + job.jobname + " Job Id: " + job.jobid +
+                                    " Spool file: " + spoolFile.ddname + "(ID #" + spoolFile.id + " Step: " +
+                                    spoolFile.stepname + ")" + "\n";
+                            }
+                        }
+                        const recordRange:string = spoolContent.response.headers['x-ibm-record-range'];
+                        const lineNumberString:string = recordRange.substring(0, recordRange.indexOf(','));
+                        startingLine = Number(lineNumberString) + 1;
+                        replyBuffer = replyBuffer + " Line " + startingLine + " : " + spoolContent.dataString;
+
+                        // If the search length is exceeded then end the search of this file.
+                        lineCount++;
+                        if(lineCount > searchLimit){
+                            Logger.getAppLogger().debug("searchJobs() - Search limit reached");
+                            startingLine = -1;
+                            replyBuffer = replyBuffer + "\n";
+                        }
+                    }
+                    else{
+                        // If nothing more is found in this file, move on to the next one.
+                        if(startingLine > 0){
+                            replyBuffer = replyBuffer + "\n";
+                        }
+                        startingLine = -1;
+                    }
+                }
+            }
+        }
+
+        return replyBuffer;
+    }
+
+    /**
+     * Get jobs (defaults to the user ID of the session as owner)
+     * @static
+     * @param {AbstractSession} session - z/OSMF connection info
+     * @param {string} searchString - The string to search for in the spool output
+     * @param {string} searchRegex - The regex to search for in the spool output
+     * @param {boolean} caseInsensitive - Specify if a search is case sensitive
+     * @param {IJobFile} jobFile - The job spool file to search
+     * @param {number} startingLine - The line to start the searching from
+     * @returns {Promise<IRestClientResponse>} - promise that resolves to spool output and response headers
+     * @memberof searchJobs
+     */
+    private static async searchSpoolContentCommon(session: AbstractSession,
+        searchString : string,
+        searchRegex:string,
+        caseInsensitive:boolean,
+        jobFile: IJobFile,
+        startingLine: number) {
+        Logger.getAppLogger().trace("SearchJobs.getSpoolContentCommon()");
```

Comment:
Same comment about 'get' vs 'search'.

Reply:
These were just a cut and paste remnant. I have corrected them.